src/App.tsx

```diff
@@ -504,5 +581,5 @@ class App extends React.Component<Props, State> {
   }
 }
 
-export default hot(connect(null, {getCoach, getUnlocked})(App));
+export default hot(connect(null, {getCoach, getUnlocked, getTraining})(App));
```

Comment:
this might be fine, but it seems odd to add a specific subject reference to the top-level App.tsx file. is there a way to generalize this?

Reply:
@thompchr getUnlocked is getting all the training data for the other tools, literacy has a different one because it is structured differently (4 types within one tool). let's discuss when we meet